api/v1/signalingserver.go

```diff
@@ -1,7 +1,8 @@
 package v1
 
-type ErrorMsg struct {
-	Error string `json:"error"`
+type Error struct {
+	Code     string `json:"code"`
+	ErrorMsg string `json:"error"`
```

Comment:
FYI: This is ok only because you kept the `error` string field. This message is sent to devices, which expect that field to be there.

Reply:
true, yes otherwise it will be an api breaking change.